deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -46,6 +46,51 @@ subjects:
 
 ---
 
+{{- if .Values.serviceAccount.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "cert-manager.serviceAccountName" . }}-tokenrequest
```

Comment:
@Jasper-Ben, is there a particular reason this name was chosen? It seems like a more natural name template for this role would be  `{{ template "cert-manager.fullname" . }}-tokenrequest`, and we now have an open PR to fix this: https://github.com/cert-manager/cert-manager/pull/7761. Any thoughts or objections?